DDM potential fraud alerts (#195)
department-of-veterans-affairs/developer-portal-content: content/direct-deposit-management/release-notes/2024-12-11.md

> Dependents are no longer allowed to make updates to direct deposit information.  This change is implemented to prevent potential data conflicts with downstream systems.

Reviewer: ```suggestion
Dependents are no longer allowed to make updates to direct deposit information. This change is implemented to prevent potential data conflicts with downstream systems.
```

> The Direct Deposit API checks the ID of the requester to determine if they are a Veteran or dependent.  If they are a dependent, or if Veteran/dependent status cannot be determined, the API generates an HTTP 422 error as notification that the requester is not allowed to make updates.  

Reviewer: ```suggestion
The Direct Deposit API checks the ID of the requester to determine if they are a Veteran or dependent. If they are a dependent, or if Veteran/dependent status cannot be determined, the API generates a HTTP 422 error to notify the requester that they are not allowed to make updates.  
```

> Access to deposit accounts flagged for fraud is not blocked by the API; rather, the HTTP alert messaging notifies when access attempts occur for flagged accounts. This puts control for blocking account access or monitoring activity with the API consumer. 

Reviewer: ```suggestion
The API doesn't block access to deposit accounts that have been flagged for fraud. Instead, it sends HTTP alerts when access attempts occur, allowing the API consumer to decide whether to block access or monitor activity.
```
